internal/integration/cli_test.go

```diff
@@ -0,0 +1,99 @@
+package integration_test
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/ionrock/procs"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	testdata "github.com/blocky/adlr/internal/integration/testdata/cli"
+)
+
+func deleteLicenseFiles() {
+	_ = os.Remove("./located-licenses.json")
+	_ = os.Remove("./identified-licenses.json")
+	_ = os.Remove("./verified-licenses.json")
+}
+
+func runADLRCLI(args string) (string, error) {
+	cmd := fmt.Sprintf("go run ../../main.go license %s", args)
+	adlrCLI := procs.NewProcess(cmd)
+	adlrCLI.OutputHandler = func(line string) string { return line }
+	adlrCLI.ErrHandler = func(line string) string { return line }
+
+	err := adlrCLI.Start()
+	if err != nil {
+		return "", fmt.Errorf("starting adlr-cli: %w", err)
+	}
+
+	_ = adlrCLI.Wait()
```

Comment:
It's often the case that we expect the command to error (e.g., couldn't locate a particular license). Thus, we wan't to ignore the error returned by Wait() and grab the commands error output to inspect in our tests